gemfiles/rails-6.0.gemfile

```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+
+gemspec path: '..'
+
+gem 'rails', '~> 6.0.0', '<= 6.1'
+gem 'sqlite3', '~> 1.3.5'
+gem 'sequel', '>= 4.0'
+gem 'bcrypt', '~> 3.1'
+
+group :test do
+  gem 'rspec-rails', '~> 3.8'
```

Comment:
I believe there must be somethjing like in Doorkeeper repo, https://github.com/doorkeeper-gem/doorkeeper/blob/master/gemfiles/rails_6_0.gemfile#L6-L10

There is a problem with Rspec 3.x and Rails 6 so we need to use Rspec master

Reply:
I've added `rspec-rails` -> `v4.0-dev`. I believe this is enough. Let me know if we need all `rspec-*` forcibly

Reply:
I don't sure it can work just with rspec updated to 4.0, so I think it would be better to have all the deps on 4.0 beta

Reply:
done it.